src/graphql/resolvers/departmentResolver.ts

```diff
@@ -0,0 +1,36 @@
+import {
+	createDepartment,
+	getDepartment,
+	updateDepartment,
+} from "../../helpers/departmentService";
+import { Arg, Int, Mutation, Query, Resolver } from "type-graphql";
+import {
+	CreateDepartmentInputModel,
+	DepartmentModel,
+	UpdateDepartmentInputModel,
+} from "../models/departmentModel";
+
+@Resolver()
+export class DepartmentResolver {
+	@Query((_returns) => [DepartmentModel])
+	async department() {
+		return getDepartment();
```

Comment:
Since this query is returning a list (multiple departments) then both the query and the service method must be plural.
i.e
"department" => "departments"
"getDepartment" => "getDepartments"